isy-konfiguration/src/main/java/de/bund/bva/isyfact/konfiguration/common/impl/ReloadablePropertyProvider.java

```diff
@@ -151,18 +151,12 @@ public synchronized boolean checkAndUpdate() {
             if (propertyDatei.isNeueVersionVerfuegbar()) {
                 LOG.info(LogKategorie.JOURNAL, EreignisSchluessel.KONFIGURATION_DATEI_NEU_GELADEN,
                     "Die Konfigurationsdatei {} wird neu geladen.", propertyDatei.getDateiname());
-                try {
                     neueVersionGeladen = true;
                     propertyDatei.neuLaden();
-                } catch (Throwable t) {
```

Comment:
.neuLaden() catches already Exceptions and throws KonfigurationDateiException.

Throwable should't not be catched

Reply:
Soll der try-catch-Block tatsächlich vollständig entfallen, oder sollte anstelle von "Throwable" nicht "Exception" gecatched werden und die Anweisungen im Catch-Block wie bisher bestehen bleiben?

Reply:
Exceptions are already caught within the neuLaden() method. No exceptions are to be expected in the normal program flow.